config.yaml

```
@@ -43,10 +43,11 @@ min_download: 10
 # This should be around 70-100% of your total download speed.
 max_download: 100
 
+manual_speed_algorithm_share: true # Set speed based on manually configured shares instead of using number of active torrents
```

```suggestion
manual_speed_algorithm_share: false # Set speed based on manually configured shares instead of using number of active torrents
```
I'd say keep the default based on torrent counts

---

I’m sorry looks like I missed that